src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -92,6 +92,7 @@ public void setAddressBookFilePath(Path addressBookFilePath) {
     @Override
     public void setAddressBook(ReadOnlyAddressBook addressBook) {
         this.addressBook.resetData(addressBook);
+        updateSelectedExamStatistics();
```

Comment:
Nice that you have abstracted it such that all new UI components call the same function